test/rails/controller_test.rb

```diff
@@ -233,6 +233,15 @@ class ActiveRecordPresentTest < ActionController::TestCase
 
     assert_equal "active_record_bands/show.html: Band, Band, true, Band::Update", response.body
   end
+
+  test "#collection" do
+    Band.destroy_all
+    Band::Create[band: {name: "Nofx"}]
+
+    get :index
+
+    assert_equal "active_record_bands/index.html: Band::ActiveRecord_Relation, Band::ActiveRecord_Relation, Band::Index", response.body
```

Comment:
squash, go, merge, refactor, edit, squash, merge, pull, branch, squash... and then one test got missed :smile_cat: